test/e2e/lib/utils.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ Utility functions used for testing
+*/
+package lib
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+// Returns the value of the environment variable if it exists, otherwise exits the test with an error
```

Comment:
The doc string is supposed to start with the name of the thing being documented, see https://blog.golang.org/godoc, "Notice this comment is a complete sentence that begins with the name of the element it describes."

How about something like this?
```suggestion
// GetEnvOrFail gets the specified environment variable. If the variable is not set, then the test exits with an error.
```